README.md

```diff
@@ -6,6 +6,8 @@ The paper comes as a single ``latex`` file ``Paper.tex``.
 
 It can be viewed in ``PDF`` format with ``pdflatex Paper.tex`` (local install of a current free tex distribution required).
 
+After creating ``Paper.pdf`` for the first time and every time the bibliography file (``Biblio.bib``) is updated, you will also need to run ``bibtex Paper`` and then ``pdflater Paper`` twice (e.g. ``bibtex Paper && pdflatex Paper && pdflatex Paper``) in order to correctly incorporate all the bibliography references.
```

Comment:
These instructions are for Ubuntu 16.04. I have no idea how to make this work on Windows or Mac OS X.